app/views/posts/index.html.erb

```diff
@@ -1 +1,15 @@
-<h1>List of Posts</h1>
\ No newline at end of file
+<h1>List of Posts</h1>
+
+<%= render 'partials/user', user: @user %>
+<ul>
+<% if @posts.empty? %>
+  <p class='no-posts'>No posts</p>
+<% else %>
+<% @posts.each do |post|%>
+<li>
+  <%= render 'partials/post', post: post %>
+  <%= render 'partials/comment', comments: post.recent_comments %>
+</li>
+<%end%>
+<%end%>
+</ul>
```

Comment:
- [ ] Kindy fix this error that is happening on the post index view, I would strongly suggest you revise the association on your models to solve this

![image](https://github.com/BANCUNGUYE66/Blog-app/assets/79058364/98374c2f-2f72-47e9-b740-6297caf14fc2)